rusoto/services/sts/src/custom/web_identity.rs

```diff
@@ -0,0 +1,198 @@
+//! The Credentials Provider to read from Environment Variables.
```

Comment:
Nit: looks to be leftover copy/paste doc comment. Update to describing the web identity provider?